generation/um/dxcapi/dxc-dxcapi.h

```diff
@@ -0,0 +1,2 @@
+#include <Windows.h>
+#include <dxcapi.h>
```

Comment:
Missing newline, also this should be named `um-dxcapi.h`
```suggestion
#include <dxcapi.h>

```